summoner-tui/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ The changelog is available [on GitHub][2].
   Improve `summon show ghc` output
 * [#255](https://github.com/kowainik/summoner/issues/255)
   Autofill custom prelude module name when package name is being typed.
+* [#253](https://github.com/kowainik/summoner/issues/253):
+  Fix not being able to add newline to multi-line fields.
```

Comment:
I believe it's not the exact wording. Let's say explicitly what it does, like "Fix new line in the Description field of the form", what do you think?

Reply:
Agreed, done :)